CharacterCreation/02_Classes.txt

```diff
@@ -112,9 +107,9 @@ Class Primary Attribute: Strength
 Starting Wealth: $4150 + 100*Luck
 
 Passives: 
-* Gain Skill Weapons - Thrown OR Weapons - Explosives at 10.
-* Gain Skill Weapons - Heavy at 10.
-* Gain Feat: Proficiency: Light Melee Weaponry.
+* Gain  10 skill points in Weapons - Thrown OR Weapons - Explosives
```

Comment:
Every class after soldier has an extra space in these lines between Gain and 10.

Reply:
The joys of copy-pasta. will fix